pkgs/tools/misc/cutecom/default.nix

```diff
@@ -1,25 +1,33 @@
-{ stdenv, fetchFromGitHub, qtbase, qtserialport, cmake }:
+{ mkDerivation, lib, fetchFromGitLab, qtbase, qtserialport, cmake, makeWrapper }:
 
-stdenv.mkDerivation rec {
+mkDerivation rec {
   pname = "cutecom";
-  version = "0.50.0";
-  src = fetchFromGitHub {
-    owner = "neundorf";
-    repo = "CuteCom";
+  version = "0.51.0";
+
+  src = fetchFromGitLab {
+    owner = "cutecom";
+    repo = "cutecom";
     rev = "v${version}";
-    sha256 = "0zjmbjrwwan9z5cphqjcq2h71cm4mw88j457lzdqb29cg4bdn3ag";
+    sha256 = "1zprabjs4z26hsb64fc3k790aiiqiz9d88j666xrzi4983m1bfv8";
   };
 
   preConfigure = ''
-    substituteInPlace CMakeLists.txt --replace "#find_package(Serialport REQUIRED)" "find_package(Qt5SerialPort REQUIRED)"
+    substituteInPlace CMakeLists.txt \
+      --replace "#find_package(Serialport REQUIRED)" "find_package(Qt5SerialPort REQUIRED)"
+  '';
+
+  buildInputs = [ qtbase qtserialport cmake makeWrapper ];
+
+  postInstall = ''
```

Comment:
makeWrapper here  is completely unneeded, `mkDerivation` automatically adds `wrapQtAppsHook` which was made for this exact purpose.

`wrapQtApp` as @jonringer has mentioned is only needed when you are in a situation where wrapping manually is needed.

Reply:
see long thread at https://github.com/NixOS/nixpkgs/issues/65399

Reply:
Thanks for this information.